src/shared/MovieList.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import useMovies from "./Api";
+
+function MovieList({ title, className }) {
```

Comment:
Nie potrzebujesz propsa className

Reply:
skasowalem, ale jak by strona miala inaczej wygladac na roznych podstronach to nadanie classy dla osobnych contenerow by sie przydalo ;)